lxd/daemon_images.go

```diff
@@ -86,10 +87,14 @@ func (d *Daemon) ImageDownload(r *http.Request, op *operations.Operation, args *
 
 		if protocol == "lxd" {
 			// Setup LXD client
-			remote, err = lxd.ConnectPublicLXD(args.Server, clientArgs)
+			server, err := lxd.ConnectLXD(args.Server, clientArgs)
```

Comment:
We should keep using `ConnectPublicLXD` and just add `UseProject` as part of the ImageServer interface instead (and have it error if passed a non-empty project in the simplestreams implementation).